data/starts.txt

```diff
@@ -0,0 +1,199 @@
+		payment -1132000
+
+# this ship uses the most expensive loadout possible (3.5M)
+# so that you +- have the same amount of value back if you sell it (1.3mill)
+# It has a small hull buff to help the player survive.
+ship "Grasshopper" "Grasshopper (Expensive)"
+	add attributes
+		"hull" 400
+	outfits
+		`"Benga" Atomic Thruster`
+		`"Biroo" Atomic Steering`
+		"Fuel Pod"
+		"Hai Chasm Batteries"
+		"Hai Corundum Regenerator"
+		"Hai Williwaw Cooling"
+		Hyperdrive
+		"Ion Cannon"
+		"Pebble Core"
+		"Pulse Rifle"
+		"Quantum Keystone" 2
+
+# the start itself cannot initiate events
+mission "Greenwater start"
+	landing
+	invisible
+	to offer
+		has "hai space start"
+	on offer
+		event "not from boston"
+		event "grew up on greenwater"
+		log "Finally scraped together enough money for a down payment on a starship. The interest on the mortgage is exorbitant."
+		log "Factions" "Republic" `Hundreds of years ago, the independent territories in different parts of human space agreed to join together into a single democratic government, with Earth as its capital. The rise of the Republic ushered in a long period of peace and prosperity in human history.`
+			`Representation in the Republic Parliament is based on population. That means that some individual "Paradise Worlds" have more representatives than entire regions of space like the Dirt Belt that are more sparsely settled.`
+		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
+			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
+		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
+		
+
+event "not from boston"
+	planet "New Boston"
+		description `Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`
+
+event "grew up on greenwater"
+	planet "Greenwater"
+		add description `It feels good to be home.`
```

Comment:
This sort of falls into the 'telling the player what they feel' issue. I would make this more similar to the New Boston start's description of New Boston.

Reply:
```suggestion
		add description `This is also the place where you grew up.`
```